packages/core/src/mm/actors/Player.S

```diff
@@ -1224,3 +1224,7817 @@ PATCH_END
 PATCH_START 0x8085e958
 .int 0
 PATCH_END
+
+/* Hammer Related ASM */
+
+PATCH_START 0x80123960
+  addiu     sp, sp, -0x20
+  sw        ra, 0x0014 (sp)
+  jal Player_CustomActionToModelGroup
+  nop
+  lw        ra, 0x0014 (sp)
+  jr        ra
+  addiu     sp, sp, 0x20
+PATCH_END
+
+PATCH_START 0x80128DDC
```

Comment:
There appears to be a branch instruction at 0x801289d8 that branches to the second line in this patch. Could that cause any issues since it skips setting `a0`?

Reply:
It does not matter because the function in question does not actually use that argument for anything. I don't remember why I set a0 at all but replacing it with a nop seemed to work ok

Reply:
In that case I'd say we should remove the `GameState_Play*` parameter from the function definition so that no one accidentally uses it in the future.